boards/acd52832/Makefile

```diff
@@ -0,0 +1,4 @@
+MODULE = board
+DIRS = $(RIOTBOARD)/common/nrf52xxxdk
```

Comment:
It's not better `+=`? (in case of this actually belongs to nrf52 common boards)

Reply:
In the current state it doesn't matter but you might be right in general. I just copied from the other nrf52-based boards. Can we take it as for the others now and open another PR in case you want to see this addressed?

Reply:
Ok, let's see how the support for this, IMHO, very complete CPU, evolves.